dev/archetypes/maven-java/src/main/resources/archetype-resources/pom.xml

```diff
@@ -52,9 +57,9 @@
                         </goals>
                         <configuration>
                             <rules>
-                                <!-- Lagom requires at least Maven version 3.2.1 -->
+                                <!-- Lagom requires at least Maven version 3.6.0 -->
                                 <requireMavenVersion>
-                                    <version>[3.2.1,)</version>
+                                    <version>[3.6.0,)</version>
```

Comment:
Since Lagom 1.5.x we document that Maven 3.6.0 is required. So updating it here.